syft/frameworks/torch/tensors/pointer.py

```diff
@@ -127,16 +128,20 @@ def __str__(self):
         """
 
         type_name = type(self).__name__
-        return (
+        out = (
             f"["
             f"{type_name} - "
-            f"id:{self.id} "
-            f"owner:{self.owner.id} "
-            f"loc:{self.location.id} "
-            f"id@loc:{self.id_at_location}"
+            # f"id:{self.id} "
+            # f"owner:{self.owner.id} "
+            f"...{str(self.id_at_location)[-3:]}@{self.location.id}"
+            # f"id@loc:"
             f"]"
         )
 
+        if(self.point_to_attr is not None):
+            out += "::" + str(self.point_to_attr)
```

Comment:
maybe you want to do `'::'.join(self.point_to_attr.split('.')`

Reply:
fixed